pkgs/applications/editors/vscode/generic.nix

```diff
@@ -135,9 +135,6 @@ let
 
       # this fixes bundled ripgrep
       chmod +x resources/app/node_modules/@vscode/ripgrep/bin/rg
-
-      # see https://github.com/gentoo/gentoo/commit/4da5959
-      chmod +x resources/app/node_modules/node-pty/build/Release/spawn-helper
```

Comment:
Doesn't seem necessary anymore? The file doesn't exist and everything seems to work fine.

Reply:
I think this is used to fix opening terminal (https://github.com/NixOS/nixpkgs/pull/214666). If that opens I guess this is fine, but this `generic.nix` file is also used by vscodium so we probably want to do a vscodium update also.